### PR TITLE
minor restructuring, adding hand-holding text

### PR DESCRIPTION
**Pull request by alecristia**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -42,44 +54,99 @@ source ~/ChildProjectVenv/bin/activate
...
-## Installing datasets
+## Getting some data
+
+You can either have some data of your own that you would like to use the package on, or you may know of some datasets that are already in this format that you'd like to reuse.
+
+It may be easier to start with an extant dataset. Here is the list that we know exists. Please note that the large majority of these data are NOT public, and thus if you cannot retrieve them, this means you need to get in touch with the data managers.
+
+### Extant datasets
+
+Instructions to download extant datasets can be found [here](http://laac-lscp.github.io/ChildRecordsData/REUSE.html).
```
Comment (lucasgautheron) [inline docs/README.md:69]: Missing so far but I guess this is intended

File: docs/README.md [hunk 2]
```diff
@@ -42,44 +54,99 @@ source ~/ChildProjectVenv/bin/activate
...
+We have prepared a public data set for testing purposes which is based on the [VanDam Public Daylong HomeBank Corpus](https://homebank.talkbank.org/access/Public/VanDam-Daylong.html); VanDam, Mark (2018). VanDam Public Daylong HomeBank Corpus. doi:10.21415/T5388S.
+
+
 
-The list of available datasets with instructions to download them can be found [here](http://laac-lscp.github.io/ChildRecordsData/PROJECTS.html).
+#### From the [LAAC team](https://lscp.dec.ens.fr/en/research/teams-lscp/language-acquisition-across-cultures)
 
-## Working with the data
+
+| Name | Authors | Location | Recordings | Audio length (hours) | Status |
```
Comment (lucasgautheron) [inline docs/README.md:80]: This is dynamic (i.e. filled with Jinja templating engine at generation time)
But README.md is static (and as the entry point for the docs on github, should remain as such in my opinion)
So everything that is dynamic should not be on the README (or just filled statically)

File: docs/templates/PROJECTS.md [hunk 3]
```diff
@@ -1,109 +0,0 @@
-- [List of available projects](#list-of-available-projects)
```
Comment (lucasgautheron) [inline docs/templates/PROJECTS.md:1]: I guess you meant to rename PROJECTS to REUSE, but instead have deleted it ?

**Review by lucasgautheron — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (alecristia) [reply docs/README.md:69]: yes, that's right -- PROJECTS relabeled to REUSE, so the html won't exist until this branch is merged with master.

File: docs/README.md (hunk 2 above)
Comment (alecristia) [reply docs/README.md:80]: roger! i'll move it then

File: docs/templates/PROJECTS.md (hunk 3 above)
Comment (alecristia) [reply docs/templates/PROJECTS.md:1]: I did rename PROJECTS as REUSE -- and I thought I corrected all mentions. Can you point out the error more clearly?

**Comment by alecristia** [comment]
I'm now incorporating these comments & other points from #84

**Pull request closed without merging**